policy/modules/services/nsd.te

```diff
@@ -34,7 +34,7 @@ files_type(nsd_zone_t)
 # Local policy
 #
 
-allow nsd_t self:capability { chown dac_override kill setgid setuid };
+allow nsd_t self:capability { chown dac_override kill setgid setuid dac_read_search net_admin };
```

Comment:
If you are netting a dac_read_search, then the dac_override probably isn't needed. Can you retest without the dac_override.

Reply:
```
 * Starting nsd ...
[2019-01-03 12:44:38.880] nsd[32656]: error: Cannot open /var/lib/nsd/nsd.log for appending (Permission denied), logging to stderr
[2019-01-03 12:44:38.880] nsd[32656]: notice: nsd starting (NSD 4.1.25)
 * start-stop-daemon: did not create a valid pid in `/run/nsd/nsd.pid'
 * Failed to start nsd                                                                                                                                                       [ !! ]
 * ERROR: nsd failed to start

# ps -ef|grep nsd
nsd      32657     1  3 12:44 ?        00:00:00 /usr/sbin/nsd -c /etc/nsd/nsd.conf
nsd      32658 32657  0 12:44 ?        00:00:00 /usr/sbin/nsd -c /etc/nsd/nsd.conf
nsd      32662 32658  0 12:44 ?        00:00:00 /usr/sbin/nsd -c /etc/nsd/nsd.conf
nsd      32663 32658  0 12:44 ?        00:00:00 /usr/sbin/nsd -c /etc/nsd/nsd.conf
nsd      32664 32658  0 12:44 ?        00:00:00 /usr/sbin/nsd -c /etc/nsd/nsd.conf
nsd      32665 32658  0 12:44 ?        00:00:00 /usr/sbin/nsd -c /etc/nsd/nsd.conf

# ls -lZ /var/lib/nsd/nsd.log
-rw-r--r--. 1 nsd nsd system_u:object_r:nsd_db_t:s0 9647 Jan  3 12:40 /var/lib/nsd/nsd.log

# sesearch -A -s nsd_t -t nsd_db_t
allow nsd_t file_type:filesystem getattr;
allow nsd_t nsd_db_t:file { append create getattr ioctl link lock map open read rename setattr unlink write };

# sesearch -A -s nsd_t -c capability
allow nsd_t nsd_t:capability { chown dac_read_search kill net_admin net_bind_service setgid setuid };
```
Without `dac_override` daemon starts, but not fully clear.
Without `dac_read_search` daemon can not start.
`dac_override` still required.